src/config/Config.zig

```diff
@@ -400,9 +400,15 @@ theme: ?Theme = null,
 /// Background color for the window.
 background: Color = .{ .r = 0x28, .g = 0x2C, .b = 0x34 },
 
+/// Background color for the titlebar.
+@"background-titlebar": ?Color = null,
+
```

Comment:
`background-titlebar` => `titlebar-background`

Precedent: `selection-background`

Reply:
Probably also need to note that this only takes effect if `window-theme=ghostty`.